raiden/transfer/state.py

```diff
@@ -388,11 +389,21 @@ class PendingWithdrawState:
     nonce: Nonce
 
 
+@dataclass
+class ConfirmedBurntByPartnerState:
+    total_burn: BurnAmount
```

Comment:
I think we need to store the complete message here, otherwise the MS cannot confirm the signature.

Reply:
ok I now saved the state-change `ReceiveBurnConfirmation` which is the authenticated message. Is this sufficient?